### ISSUE TITLE
Compilation failure "error: lvalue required as decrement operand"

### ISSUE THREAD

**Issue by jbigot** [pdidev/pdi#432]
https://github.com/pdidev/test_env/actions/runs/3457460311/jobs/5777149488

**Comment by jbigot** [comment]
```
ext/_yaml.c:23892:5: error: lvalue required as decrement operand
  --Py_REFCNT(o);
  ^~
```

**Comment by jbigot** [comment]
Fixed in 59215e8443d3c8b3d9c699eafb1e96c523ef5923 by upgrading PyYaml
